src/hypertrace/agent/autoinstrumentation/sitecustomize.py

```diff
@@ -0,0 +1,48 @@
+'''Enable instrumentationon all supported modules.''' # pylint: disable=R0401
+import os
+import logging
+from hypertrace.agent import Agent
+
+DEFAULTS = [
+    'flask',
+    'mysql',
+    'postgresql',
+    'grpc:server',
+    'grpc:client',
+    'requests',
+    'aiohttp'
+]
+
+# Initialize logger
+logger = logging.getLogger(__name__)  # pylint: disable=C0103
+
+MODULES = ''
+if 'HT_INSTRUMENTED_MODULES' in os.environ:
+    logger.debug("[env] Loaded HT_INSTRUMENTED_MODULES from env")
+    MODULES = os.environ['HT_INSTRUMENTED_MODULES']
```

Comment:
can we remove spaces in here?

Reply:
It isn't clear which spaces you are referring to.

Reply:
I think you mean spaces from the HT_INSTRUMENTED_MODULES list. This is now done.